app/controllers/accounts_controller.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
 class AccountsController < ApplicationController
+  before_action :set_account, only: %i[edit update archive resend_confirmation enable_dashboard disable_dashboard]
```

Comment:
@hengsokly I would suggest to use term authorize_account instead of set_account as it reflect to the main behavior of the method